src/services/SnapshotService.ts

```diff
@@ -81,4 +81,20 @@ export class SnapshotService {
       }
     })
   }
+
+  static async getSnapshotStatusAndSpace(spaceName?: string) {
```

Comment:
Could you add a default here so line 86 can be removed?
`getStatusAndSpace(spaceName = SNAPSHOT_SPACE)`